### DISCUSSION TITLE
linking IP address to device interface - not easy to find interface

### DISCUSSION THREAD

**Discussion by carlo-4ip** [netbox-community/netbox#12606] [Ideas]
Hello,

I noticed that in version 3.5.1 the "device" field is gone when creating an IP address object.

It would make life a lot easier if this field is returned to the form.

Also, is it possible to only show the devices belonging to the selected Tenant? And then only show the interfaces belonging to the selected device?

**Comment by candlerb** [comment]
> I noticed that in version 3.5.1 the "device" field is gone when creating an IP address object.

"Device" was a filter to narrow down the interfaces to one particular device. Now you have to click the little search button on the right to get to it.

![image](https://github.com/netbox-community/netbox/assets/44789/3b920812-24f0-4974-857f-5113d1c7283e)

This has been raised before as making the interface a bit more clunky: see #12577. And AFAICS, you can only filter on device name, not device tenant.

**Reply by carlo-4ip** [reply]
Thank you for your response. I saw that search field, but that's very extensive. We only really need the devices, so I was sad to see the old field was gone.

I did not find that other thread. Thank you for that, I will close this one and join the other one.
